packages/broker/src/plugins/operator/OperatorPlugin.ts

```diff
@@ -45,10 +49,21 @@ export class OperatorPlugin extends Plugin<OperatorPluginConfig> {
             this.streamrClient,
             toEthereumAddress(this.pluginConfig.operatorContractAddress)
         )
+        const operatorFleetState = new OperatorFleetState(
+            this.streamrClient,
+            toStreamID('/operator/coordination', serviceHelperConfig.operatorContractAddress)
+        )
         this.maintainTopologyService = new MaintainTopologyService(
             this.streamrClient,
-            new MaintainTopologyHelper(
-                serviceHelperConfig
+            new StreamAssignmentLoadBalancer(
+                'myNodeId', // TODO: real nodeId here
```

Comment:
Can we get the real nodeId synchronously, or do we need something like `(await streamrClient.getNode()).getNodeId()`?

Maybe it would make sense to move this the load balancer initialization to into `MaintainTopologyService#start `anyway?